aspnet/mvc/overview/older-versions-1/nerddinner/index.md

```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions-1/nerddinner/index
 title: "NerdDinner | Microsoft Docs"
 author: rick-anderson
-description: "How to build the NerdDinner.com website using ASP.NET MVC 1. For the ASP.MVC 3 version, please visit nerddinner.codeplex.com ."
+description: "How to build the NerdDinner.com website using ASP.NET MVC 1. For the ASP.NET MVC 3 version, visit nerddinner on GitHub ."
```

Comment:
Also, please remove the extra space before the period at the end of the sentence.